ninja_ide/translations.py

```diff
@@ -1,885 +1,885 @@
-TR_DO_YOU_WANT_TO_REMOVE = tr("NINJA-IDE", "Do you want to remove it?")
-TR_LOAD_DEFAULTS = tr("NINJA-IDE", "Load Defaults")
-TR_SHORTCUTS_ARE_GOING_TO_BE_REFRESH = tr(
-    "NINJA-IDE",
-    "Menu item shortcuts will be refreshed on restart.")
-TR_NOTIFICATION = tr("NINJA-IDE", "Notification")
-TR_POSITION_ON_SCREEN = tr("NINJA-IDE", "Position On Screen")
-TR_LEFT = tr("NINJA-IDE", "left")
-TR_RIGHT = tr("NINJA-IDE", "right")
-TR_TOP = tr("NINJA-IDE", "top")
-TR_BOTTOM = tr("NINJA-IDE", "bottom")
-
-# Preferences Execution
-TR_SELECT_SELECT_PYTHON_EXEC = tr("NINJA-IDE", "Select Python executable")
-TR_SELECT_PYTHON_EXEC = tr("NINJA-IDE", "Python Exec:")
-TR_WORKSPACE_PROJECTS = tr("NINJA-IDE", "Workspace and Project:")
-TR_PYTHON_OPTIONS = tr("NINJA-IDE", "Execution Options")
-TR_SELECT_EXEC_OPTION_B = tr(
-    "NINJA-IDE",
-    "-B: don't write .py[co] files on import")
-TR_SELECT_EXEC_OPTION_D = tr("NINJA-IDE", "-d: debug output from parser")
-TR_SELECT_EXEC_OPTION_E = tr(
-    "NINJA-IDE",
-    "-E: ignore PYTHON* environment variables (such as PYTHONPATH)")
-TR_SELECT_EXEC_OPTION_O = tr(
-    "NINJA-IDE",
-    "-O: optimize generated bytecode slightly")
-TR_SELECT_EXEC_OPTION_OO = tr(
-    "NINJA-IDE",
-    "-OO: remove doc-strings in addition to the -O optimizations")
-TR_SELECT_EXEC_OPTION_Q = tr("NINJA-IDE", "-Q: division options:")
-TR_SELECT_EXEC_OPTION_s = tr(
-    "NINJA-IDE",
-    "-s: don't add user site directory to sys.path")
-TR_SELECT_EXEC_OPTION_S = tr(
-    "NINJA-IDE",
-    "-S: don't imply 'import site' on initialization")
-TR_SELECT_EXEC_OPTION_T = tr(
-    "NINJA-IDE",
-    "-t: issue warnings about inconsistent tab usage")
-TR_SELECT_EXEC_OPTION_TT = tr(
-    "NINJA-IDE",
-    "-tt: issue errors about inconsistent tab usage")
-TR_SELECT_EXEC_OPTION_V = tr(
-    "NINJA-IDE",
-    "-v: verbose (trace import statements)")
-TR_SELECT_EXEC_OPTION_W = tr("NINJA-IDE", "-W: warning control:")
-TR_SELECT_EXEC_OPTION_X = tr("NINJA-IDE", "-x: skip first line of source")
-TR_SELECT_EXEC_OPTION_3 = tr(
-    "NINJA-IDE",
-    "-3: warn about Python 3.x incompatibilities "
-    "that 2to3 cannot trivially fix")
-
-
-# 2to3
-TR_CURRENT_CODE = tr("NINJA-IDE", "Current code")
-TR_SUGGESTED_CHANGES = tr("NINJA-IDE", "Suggested code")
-TR_APPLY_CHANGES = tr("NINJA-IDE", "Apply changes")
-TR_SAVE_BEFORE_APPLY = tr("NINJA-IDE", "Save the file before applying changes")
-
-
-# from_import_dialog
-TR_INSERT_AT_LINE = tr("NINJA-IDE", "Insert at line")
-TR_WITH_COMMENT = tr("NINJA-IDE", "with comment")
-TR_ADD = tr("NINJA-IDE", "Add")
-
-
-# Themes
-TR_CODE = tr("NINJA-IDE", "Code")
-TR_PREVIEW = tr("NINJA-IDE", "Preview")
-TR_SCHEME_INVALID_NAME = tr(
-    "NINJA-IDE",
-    "The name you have chosen is invalid. "
-    "\nPlease pick a different name.")
-TR_WANT_OVERWRITE_FILE = tr("NINJA-IDE", "Do you want to overwrite the file")
-TR_SCHEME_SAVED = tr("NINJA-IDE", "The scheme has been saved")
+# -*- coding: utf-8 -*-


```

Comment:
What is the symbol added in the file? Could you remove that? It like that you change all the file, but is the same.